packages-exp/auth-exp/src/model/enum_maps.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * An enum of factors that may be used for multifactor authentication.
+ *
+ * @public
+ */
+export const FactorId = {
+  /** Phone as second factor */
+  PHONE: 'phone'
+};
```

Comment:
Is it worth declaring `{...} as const`? This will make `PHONE` typed as `"PHONE"` (string literal) instead of string, which may be a bit more specific. Ditto for below.

Reply:
Good idea, done